### DISCUSSION TITLE
Decap CMS is Died?

### DISCUSSION THREAD

**Discussion by shineholiday** [decaporg/decap-cms#6841] [General]
I see that DecapCMS hasn't made any updates,
Just changing the name, but not changing anything else?
What's the purpose of the rebrand?

**Comment by jessevanmuijden** [comment]
https://discord.com/channels/1078661770119487600/1078661770119487603/1128756492930121820

**Comment by martinjagodic** [comment]
@shineholiday yes, currently the CMS is only rebranded, but there is work in progress that will bring new life to the project. You can review the PR here: https://github.com/decaporg/decap-cms/pull/6852
